.github/workflows/security-scan.yml

```
@@ -0,0 +1,20 @@
+      - '**/package-lock.json'
+      - '**/requirements.txt'
+
+jobs:
+  security-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Vulnerability Scanner
+        uses: fylein/vulnerability-scan-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          is_submodule: true 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)